HTML/Vehicles.html

```diff
@@ -1,8 +1,128 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
 <head>
-</head>
-<body bgcolor = "violet">
-<h1> This is the Vehicles page </h1>
+    <style>
+        .pageHeader{
+            text-align: center;
+        }
+        .carRow{
+            display: flex;

```

Comment:
For centering you could also take advantage of flex box properties. Such as align-items to center vertically and justify-content to center horizontally. Margin auto is good from experience it's usually never works with some additional properties. align-items: center; justify-content: center should work and it does not break as easily.

Reply:
ah ok, I'll change that then

Reply:
Yeah its pretty cool once you know about flexbox I use it all the time. There's also display: grid but that is very tricky to use in my opinion